t/admin/ssl4.t

```diff
@@ -242,7 +242,6 @@ apisix:
             - qeddd145sfvddff3
 --- error_log
 decrypt ssl key failed
-[alert]
```

Comment:
Because the error generated by decryption has been cleared, ngx will not throw this error.